bin/diagnostic.js

```diff
@@ -1 +1,16 @@
 'use strict';
+
+
+const ask = require('../lib/ask.js');
+
+let answer = 'not empty';
+
+while (answer !== '') {
+  answer = ask("Guess my secret? ");
+  if (answer === '9') {
+   console.log("You guessed it!");
+   else if (answer <= 1 || answer >= 10);
+   console.log("Guess beween 1 and 10")
+   break;
+  }
+}
```

Comment:
Very close. Your strategy is right on target, but some of the details are off:
1) Make sure that your brackets match. If you are missing a `{` or a `}`, your code won't run properly. Easiest way to do that is to run your code.
2) `break;` means to get out of the loop. You have in the clause that says, "Guess between 1 and 10". So when I guessed 89, I got a message saying "Guess between 1 and 10", and then the program quit. 
3) The question said, "If `answer` isn't >= 1 and <= 10" - so 1 and 10 are _included_ in the values we CAN accept. But what you have on line 12 is `if (answer <= 1 || answer >= 10)`.